ui/src/views/compute/InstanceTab.vue

```diff
@@ -33,7 +33,7 @@
         <router-link :to="{ path: '/iso/' + vm.isoid }">{{ vm.isoname }}</router-link> <br/>
         <barcode-outlined /> {{ vm.isoid }}
       </a-tab-pane>
-      <a-tab-pane :tab="$t('label.volumes')" key="volumes">
+      <a-tab-pane :tab="$t('label.volumes')" key="volumes" v-if="'listVolumes' in $store.getters.apis">
```

Comment:
I think, it's better to the Volumes tab and show empty list when no permissions.

Reply:
The other tabs in this page are hidden when the user does not have permission to list their contents. I don't see why the volumes tab should be an exception.

Showing an empty list may lead users into thinking that the instance does not have any volumes.